src/SFA.DAS.EmployerFinance.Web/Controllers/HomeController.cs

```diff
@@ -22,11 +27,12 @@ public IActionResult Index()
             return View();
         }
 
-        [Route("error")]
+        [AllowAnonymous]
+        [Route("error/{code}")]
         [ResponseCache(Duration = 0, Location = ResponseCacheLocation.None, NoStore = true)]
-        public IActionResult Error()
+        public IActionResult Error(int code)
         {
-            return View(new ErrorViewModel { RequestId = Activity.Current?.Id ?? HttpContext.TraceIdentifier });
+            return View(this.ViewExists(_viewEngine, $"Errors/{code}") ? $"Errors/{code}" : "Errors/Error");
```

Comment:
I recommend rendering a static error page instead of executing another request though MVC's pipeline.